components/builder-api-client/src/lib.rs

```diff
@@ -268,14 +262,12 @@ pub trait DisplayProgress: Write {
 pub struct Client(ApiClient);
 
 impl Client {
-    pub fn new<U>(
-        endpoint: U,
-        product: &str,
-        version: &str,
-        fs_root_path: Option<&Path>,
-    ) -> Result<Self>
-    where
-        U: IntoUrl,
+    pub fn new<U>(endpoint: U,
+                  product: &str,
+                  version: &str,
+                  fs_root_path: Option<&Path>)
+                  -> Result<Self>
+        where U: IntoUrl
     {
```

Comment:
I think this is also `indent_style = "Visual"` though, and in this case, I think it's a big improvement.